docs/logfire-costs.md

```diff
@@ -0,0 +1,95 @@
+# Logfire Billing & Usage Guide
+
+## Exactly How Logfire Charges Work
+
+* **What we meter:** every span, log **or** metric you ship. If you're not sure what those are, check out
+our [concepts page](concepts.md)
```

Comment:
the concept page doesn't mention metrics at all, and it doesn't really explain what logs are either

Reply:
@summerscope will you include this in your polish of concepts?